shared/js/dataDriver.js

```diff
@@ -7,6 +7,22 @@ loop.DataDriver = function() {
   "use strict";
 
   let { actions } = loop.shared;
+  /*
+   * When creating new schemas, be sure to add them to _getSchemaByType method
+   */
+  const CHAT_VALIDATOR_SCHEMA = {
+    timestamp: Number,
+    value: Object
+  };
+
+  const PARTICIPANT_VALIDATOR_SCHEMA = {
+    value: Object
```

Comment:
Any reason no timestamp here? All schemas have to be one level deep for Validator, and all records should have at most two fields of timestamp and optionally value (for deleted records).